index.html

```diff
@@ -1,192 +0,0 @@
-        quos
-        dignissimos voluptate, nobis laborum laboriosam, hic veritatis amet perferendis dolore repellat incidunt unde
-        delectus magnam placeat soluta maxime illo ipsam voluptatum. Fuga explicabo veritatis quasi esse quibusdam
-        reprehenderit repudiandae aliquam sequi quo accusantium, id aliquid? Excepturi, non! Cupiditate iusto
-        necessitatibus
-        porro. Earum ut eum voluptatem mollitia ad, deserunt eveniet quae inventore corrupti dignissimos delectus
-        voluptas,
-        id fugit sit perspiciatis labore. Natus id, atque error soluta itaque quam nesciunt vero porro maxime nihil quas
-        voluptatibus.</h4>
-      </p>
-      <p>
-      <h4>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Minima perferendis veritatis maxime error. Debitis
-        voluptatum ullam doloremque suscipit vitae explicabo? Enim quidem adipisci, omnis delectus atque laboriosam
-        optio
-        fugiat maxime saepe aliquid. Non delectus expedita facere reprehenderit at repellendus hic recusandae eveniet
-        minus
-        autem assumenda, distinctio tempore pariatur dolore! Eius, illum quia. Suscipit veritatis iure dolorum molestias
-        amet voluptatum aut facilis odio doloremque est voluptas incidunt ipsum id maxime vitae, ad quisquam porro
-        architecto soluta laborum, reprehenderit numquam? Deserunt voluptate ab maxime unde eius modi dolorem similique
-        non
-        doloribus voluptas!</h4>
-      </p>
-      <button><a href="./continue-reading.html">Continue Reading</a></button>
-    </section>
-
-    <hr>
-    <hr>
-    <!-- something of note with image and text float  -->
-    <section id="bottom">
-      <h3><a href="./continue-reading.html">Something Of Note</a></h3>
-      <img src="./images/fusion.jpg" alt=" " />
-      <h4>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Et iure qui tempora accusantium rerum voluptatem
-        molestias tenetur deleniti iusto provident, minima dicta, atque obcaecati laboriosam, est ratione adipisci
-        pariatur
-        quos nihil nostrum impedit suscipit nisi! Provident amet dolor eaque pariatur id! Repudiandae asperiores
-        cupiditate
-        pariatur.</h4>
-      <button><a href="./learn-more.html">Learn more</a></button>
-
-      <hr>
-
-
-      <h3><a href="./continue-reading.html">Something Of Less Note</a></h3>
-      <img src="./images/blackgoku.jpg" alt=" " />
-      <h4>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorem maxime dolor vero modi! Voluptatem dolores
-        tempore illum dolore dignissimos, eum, ipsam corporis blanditiis ex ab id! Porro, aliquid? Cum numquam sit
-        itaque,
-        nihil quas blanditiis. Placeat, facilis. Illum amet, ullam rerum quam et sed quas.</h4>
-      <button><a href="./learn-more.html">Learn more</a></button>
-    </section>
-  </section>

```

Comment:
This whole section is within a large container, this is totally fine, however, breaking up this section into more smaller containers will make it much easier to lay things out, you wont have to account for all the different sections at once, basically it helps to break things down as much as possible into bite sized chunks. 
Per the other comment, try wrapping some of these interior containers in their own sections, then styling each individual section as you go so there isn't as much to keep track of as once